test/functional/local_transaction_controller_test.rb

```diff
@@ -162,13 +140,17 @@ class LocalTransactionControllerTest < ActionController::TestCase
 
       context "for electoral registration for a Northern Ireland local authority" do
         setup do
-          stub_mapit_has_a_postcode_and_areas(
+          stub_locations_api_has_location(
             "BT1 3QG",
-            [0, 0],
             [
-              { "name" => "Belfast City Council", "type" => "LGD", "govuk_slug" => "belfast", "country_name" => "Northern Ireland" },
+              {
+                "latitude" => 0,
+                "longitude" => 0,
```

Comment:
I don't think these values are used, are they? 
Can we remove them and have `stub_locations_api_has_location` set defaults for latitude/longitude if not provided?

Reply:
No, we don't use anywhere these coordinates.